crates/starknet-devnet-server/src/rpc_core/error.rs

```diff
@@ -22,10 +22,16 @@ impl RpcError {
     }
 
     /// Creates a new `ParseError`
-    pub const fn parse_error() -> Self {
-        Self::new(ErrorCode::ParseError)
+    pub fn parse_error<R>(reason: R) -> Self
+    where
+        R: Into<String>,
+    {
+        RpcError {
+            code: ErrorCode::ParseError,
+            message: Cow::Borrowed(ErrorCode::ParseError.message()),
+            data: Some(serde_json::json!({ "reason": reason.into() })),
```

Comment:
The `json` macro is already imported, can be used directly. Applicable to L53, too.